pkg/helpers/aws.go

```diff
@@ -99,6 +99,11 @@ func CloudformationInstall(cf cloudformationiface.CloudFormationAPI, name, templ
 		TemplateURL: aws.String(template),
 	}
 
+	sadown, saup := params["ScheduledActionDownCron"], params["ScheduledActionUpCron"]
+	if (sadown == "" || saup == "") && (sadown != "" || saup != "") {
```

Comment:
What do you think about adding this validation to https://github.com/convox/rack/blob/master/pkg/cli/rack.go#L114
And the update one to https://github.com/convox/rack/blob/master/pkg/cli/rack.go#L233.

I don't think we need to go much further on the code if the params are incorrectly set.